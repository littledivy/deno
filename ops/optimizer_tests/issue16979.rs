fn op_stat_sync(
  state: &mut OpState,
  path: &str,
  lstat: bool,
  out_buf: &mut [u32],
) -> Result<(), AnyError> {}
