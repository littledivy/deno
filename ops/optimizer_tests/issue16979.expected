=== Optimizer Dump ===
returns_result: true
has_ref_opstate: true
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: true
fast_result: Some(Void)
fast_parameters: [V8Value, SeqOneByteString, Bool, Uint32Array]
transforms: {1: Transform { kind: SeqOneByteString(Ref), index: 1 }, 3: Transform { kind: SliceU32(true), index: 3 }}
is_async: false
fast_compatible: true
